tests/func/utils/test_strict_yaml.py

```diff
@@ -20,7 +20,7 @@
   3 │   cmd: python train.py
 
 Found duplicate key "cmd" with value "python train.py" (original value:\
```

Comment:
Does this fit on a single line now?
```suggestion
Found duplicate key "cmd" with value "python train.py" (original value: "python
```

Reply:
Probably linters will complain though, black does not for some reason, hmm.